SmartDeviceLink/SDLChoiceSetManager.m

```diff
@@ -160,33 +167,35 @@ - (void)sdl_updateTransactionQueueSuspended {
 #pragma mark - State Management
 
 - (void)didEnterStateShutdown {
```

Comment:
Do you want me to fix this issue in other screen sub-managers or should I open another issue?

Reply:
I looked at the other sub managers and it don't think it is an issue in those managers. The problem in the choice set manager is that it can take a few seconds to send 100 choice items serially so if the transport is destroyed while the items are being uploaded, it is possible for the choice set upload to complete after the manager has been stopped.

There is a small, but unlikely chance that one of the class properties could be reset in the Text and Graphics manager after shutdown. Since there is no state machine in that class, I simply reset the properties in the `start` method.